code/frontend/MakeItSo/MakeItSo/MakeItSo.docc/Tutorials/Building a simple todo list UI.tutorial

```diff
@@ -0,0 +1,473 @@
+@Tutorial(time: 25, projectFiles: building_the_basic_ui.zip) {
+  @XcodeRequirement(title: "Xcode 14", destination: "https://xcodereleases.com/")
+  @Intro(title: "Building a Simple Todo List UI") {
+    This tutorial guides you through building _Make It So_, a todo list application. You will start by creating s simple UI for displaying todo list items.
+    
+    @Image(source: building-todo-ui-intro.png, alt: "")
+  }
+  
+  @Section(title: "Create a New Project") {
+    @ContentAndMedia {
+      Create a new Xcode project that uses SwiftUI. 
+      
+      To preview and interact with views from the canvas in Xcode, and to use all the latest features described throughout the tutorials, ensure your Mac is running macOS Monterey or later.
+      
+      @Image(source: placeholder-image.png, alt: "")
+    }
+    
+    @Steps {
+      @Step {
+        Open Xccode and either click _Create a new Xcode project_ in Xcode's startup window, or choose _File > New > Project_.
+        
+        @Image(source: building-todo-ui-xcode-new-project.png, alt: "")
+      }
+      
+      @Step {
+        In the template selector, select _iOS_ as the platform, select the _App_ template, and then click _Next_.
+        
+        @Image(source: building-todo-ui-xcode-app-template.png, alt: "")
+      }
+      
+      @Step {
+        Select _SwiftUI_ from the Interface pop-up menu and _SwiftUI App_ from the Life Cycle pop-up menu, then click _Next_. Choose a location to save the `MakeItSo` project on your Mac.
```

Comment:
When I was following the steps, I couldn't see a "Life Cycle pop-up menu" (the screenshot in the tutorial also does not contain such option).

Reply:
You're right, this is now called "Interface", as can be seen in the screenshot. I copied the text from Apple's tutorial, hence it's outdated.